x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/action/EsqlCapabilities.java

```diff
@@ -92,6 +92,7 @@ private static Set<String> capabilities() {
 
         if (Build.current().isSnapshot()) {
             caps.add(LOOKUP_COMMAND);
+            caps.add(TABLES_TYPES);
```

Comment:
nit: I'd argue that we should actually remove `LOOKUP_COMMAND`, as `TABLES_TYPES` introduced a breaking change and should therefore be treated as a `LOOKUP_COMMAND_V2`.

If we do this, we should align the yaml tests as well - these currently require both `tables_types` AND `lookup_command` and would be skipped if we removed `LOOKUP_COMMAND`.

For our test suites, it should make no difference if we have both caps here or only `TABLES_TYPES`; however, having both capabilities is like a node was saying "I can handle both the old and the new lookup!", which is wrong and could lead to failures in CCQ-scenarios where the remote cluster is _newer_ than the local cluster.